recipe/conda_forge_ci_setup/upload_or_check_non_existence.py

```diff
@@ -14,6 +15,17 @@
 from conda_build.conda_interface import subdir as conda_subdir
 from conda_build.conda_interface import get_index
 import conda_build.api
+import conda_build.config
+
+
+def split_pkg(pkg):
+    if not pkg.endswith(".tar.bz2"):
+        raise RuntimeError("Can only process packages that end in .tar.bz2")
+    pkg = pkg[:-8]
+    plat, pkg_name = pkg.split("/")
```

Comment:
This might not work on windows boxes. Most likely should use actual os path tools.